lib/phx_tools_web/live/phx_tools_live/index.html.heex

```diff
@@ -1,36 +1,44 @@
-<div class="solved-height">
-  <div class="min-w-full min-h-full flex items-center justify-center">
-    <div class="block mt-[5%]">
-      <h1 class="text-white text-center lg:text-[24px] md:text-[20px] sm:text-[18px] mb-[10%]">
-        Choose your operating system
-      </h1>
-      <div class="flex justify-between lg:px-4 md:px-8 sm:px-10">
-        <.link navigate={~p"/macOS"}>
-          <div
-            id="macOS"
-            class={if(@operating_system == "Mac" , do: "bg-[#322199] ", else: "") <> "block border-[2px]
-            border-[#5337FF] px-3 py-2 rounded-md cursor-pointer hover:bg-[#322199]"}
-          >
-            <img src={~p"/images/macos.png"} />
-            <h1 class="text-white text-center">macOS</h1>
-          </div>
-        </.link>
-        <.link navigate={~p"/linux"}>
-          <div
-            id="linux"
-            class={if(@operating_system == "Linux", do: "bg-[#322199] ", else: "") <> "block border-[2px]
-            border-[#5337FF] px-1 py-2 rounded-md cursor-pointer hover:bg-[#322199]"}
-          >
-            <img src="/images/linux.png" />
-            <h1 class="text-white text-center">Linux</h1>
-          </div>
-        </.link>
-      </div>
-      <%= if @operating_system do %>
-        <h1 class="text-center text-white text-[20px] mt-[20%]">
-          Confirm your choice by clicking
+<%= if @page do %>
+  <InstructionComponents.render page={@page} operating_system={@operating_system} url={@url} />
```

Comment:
Live components have `render` functions. Functional components have descriptive names and several of them can appear in one module.

> It is typically best to group related functions into a single module, as opposed to having many modules with a single render/1 function. Function components support other important features, such as slots. You can learn more about components in [Phoenix.Component](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#content).

https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#sigil_H/2-heex-extension-defining-function-components